doc/examples/plot_line_hough_transform.py

```diff
@@ -64,70 +64,71 @@
 import numpy as np
 import matplotlib.pyplot as plt
 
-# Construct test image
-
+# Constructing test image.
 image = np.zeros((100, 100))
-
-
-# Classic straight-line Hough transform
-
 idx = np.arange(25, 75)
 image[idx[::-1], idx] = 255
 image[idx, idx] = 255
 
+# Classic straight-line Hough transform.
 h, theta, d = hough_line(image)
 
-fig, (ax1, ax2, ax3) = plt.subplots(1, 3, figsize=(8,4))
+# Generating figure 1.
+fig, (ax1, ax2, ax3) = plt.subplots(1, 3, figsize=(12, 6))
+plt.tight_layout()
 
-ax1.imshow(image, cmap=plt.cm.gray)
+ax1.imshow(image, cmap=cm.gray)
 ax1.set_title('Input image')
 ax1.set_axis_off()
 
-ax2.imshow(np.log(1 + h),
-             extent=[np.rad2deg(theta[-1]), np.rad2deg(theta[0]),
-                     d[-1], d[0]],
-             cmap=plt.cm.gray, aspect=1/1.5)
+ax2.imshow(np.log(1 + h), extent=[np.rad2deg(theta[-1]), np.rad2deg(theta[0]),
+           d[-1], d[0]], cmap=cm.gray, aspect=1/1.5)
 ax2.set_title('Hough transform')
 ax2.set_xlabel('Angles (degrees)')
 ax2.set_ylabel('Distance (pixels)')
 ax2.axis('image')
 
-ax3.imshow(image, cmap=plt.cm.gray)
-rows, cols = image.shape
+ax3.imshow(image, cmap=cm.gray)
+row1, col1 = np.shape(image)
 for _, angle, dist in zip(*hough_line_peaks(h, theta, d)):
     y0 = (dist - 0 * np.cos(angle)) / np.sin(angle)
-    y1 = (dist - cols * np.cos(angle)) / np.sin(angle)
-    ax3.plot((0, cols), (y0, y1), '-r')
-ax3.axis((0, cols, rows, 0))
+    y1 = (dist - col1 * np.cos(angle)) / np.sin(angle)
+    ax3.plot((0, col1), (y0, y1), '-r')
+ax3.axis((0, col1, row1, 0))
 ax3.set_title('Detected lines')
 ax3.set_axis_off()
 
-# Line finding, using the Probabilistic Hough Transform
-
+# Line finding using the Probabilistic Hough Transform.
 image = data.camera()
 edges = canny(image, 2, 1, 25)
 lines = probabilistic_hough_line(edges, threshold=10, line_length=5,
                                  line_gap=3)
 
-fig, (ax1, ax2, ax3) = plt.subplots(1, 3, figsize=(8,4), sharex=True, sharey=True)
+# Generating figure 2.
+fig, (ax1, ax2, ax3) = plt.subplots(1, 3, figsize=(16, 6), sharex=True,
+                                    sharey=True)
+plt.tight_layout()
 
-ax1.imshow(image, cmap=plt.cm.gray)
+ax1.imshow(image, cmap=cm.gray)
 ax1.set_title('Input image')
 ax1.set_axis_off()
 ax1.set_adjustable('box-forced')
 
-ax2.imshow(edges, cmap=plt.cm.gray)
+ax2.imshow(edges, cmap=cm.gray)
 ax2.set_title('Canny edges')
 ax2.set_axis_off()
 ax2.set_adjustable('box-forced')
 
-ax3.imshow(edges * 0)
-
+ax3.imshow(edges*0)
```

Comment:
Hi Emmanuelle,
I found the answer on this one. According to PEP-8, spaces on operations can be used to separate operations with different precedence, such as `sum(a*b + c*d)`. I believe we don't need the spaces when we have only one operation happening.